application/browser/application_system_tizen.cc

```diff
@@ -93,21 +124,12 @@ bool ApplicationSystemTizen::LaunchFromCommandLine(
   base::FilePath exec_path(args[0]);
 
   std::string app_id = exec_path.BaseName().MaybeAsASCII();
-  LOG(INFO) << "Attempting to launch the app with id: " << app_id;
-  if (!IsValidApplicationID(app_id))
-    return false;
-
-  ApplicationServiceTizen* app_service_tizen =
-      ToApplicationServiceTizen(application_service_.get());
-
-  Application* app = app_service_tizen->LaunchFromAppID(app_id);
-
-  if (!app)
-    return false;
 
   const std::string& name = std::string("xwalk-") + app_id;
   appcore_ops.cb_app = application_event_cb;
-  appcore_ops.data = app;
+  appcore_ops.data =
+      new AppcoreHandlerData(this,
+          base::FilePath(cmd_line.argv()[0]).BaseName().MaybeAsASCII());
```

Comment:
we have already "std::string app_id = exec_path.BaseName().MaybeAsASCII();" , so why not
new AppcoreHandlerData(this, app_id ) ?